.storybook/main.js

```diff
@@ -1,29 +1,66 @@
-const NodePolyfillPlugin = require("node-polyfill-webpack-plugin")
+const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
+
+// FIXME SW Delete this constant once every story has been fixed following the service worker removal
+const WORKING_STORIES = [
```

Comment:
Since you've declared this constant as "working", we could extract the non working in their own variable
Or just name the constant `STORIES`

Reply:
It's just a temporary list where broken stories are commented so you just have to uncomment them and start fixing. 
As the `FIXME SW` comment above implies, it will never make its way into main. But I've rename it into `STORIES` anyway so that we can go forward. Thanks for the suggestion!